src/uMongoGridfs.pas

```diff
@@ -102,11 +102,17 @@ function TMongoGridfs.FindFile(const AName: UTF8String;
   nativeFile: Pointer;
   err: bson_error_t;
 begin
+  FillChar(err, SizeOf(bson_error_t), 0);
   nativeFile := mongoc_gridfs_find_one_cnv_by_filename(FNativeGridfs,
                                                        PAnsiChar(AName),
                                                        @err, FlagsToNative(AFlags));
   if nativeFile = nil then
```

Comment:
@alexeyvo , can you move this code to a private procedure?
It looks exactly the same as in line starting in 150 ( https://github.com/Convey-Compliance/mongoc-wrapper-delphi/pull/18/files#diff-dfdcf99b8bd786acbcb599a61e80daeaR150 )